cves/CVE-2013-1965.yml

```diff
@@ -205,4 +227,13 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those in the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    The problem which caused this vulnerability was one of design. The conclusion 
+    eventually reached was that OGNL evaluation in XWork core simply cannot easily 
```

Comment:
Why were they initially performing OGNL evaluation? Might be nice to include a little context about why they opted to use this technology initially in this answer, if you can find it

Reply:
Originally, XWork core was a separate repository, and OGNL evaluation was done there.  This vulnerability was migrated with XWork core to this repository, and history predating that migration seems to be mostly wiped clean.